sphinx/source/docs/user_guide/examples/extensions_example_ticking.py

```diff
@@ -1,29 +1,30 @@
 from bokeh.io import show
 from bokeh.models import TickFormatter
 from bokeh.plotting import figure
+from bokeh.util.compiler import TypeScript
 
-JS_CODE = """
-import {TickFormatter} from "models/formatters/tick_formatter"
+TS_CODE = """
+import { TickFormatter } from "models/formatters/tick_formatter"
 
-export class MyFormatter extends TickFormatter
+export class MyFormatter extends TickFormatter{
   type: "MyFormatter"
```

Comment:
This is valid code on the type-level, just not at the run time. Should be `type = "MyFormatter"`, but preferably defined on the `prototype` (in the other case).